lib/hammer_cli_foreman/associating_commands.rb

```diff
@@ -73,7 +72,6 @@ class RemovePuppetEnvironmentCommand < HammerCLIForeman::RemoveAssociatedCommand
         success_message _("The environment has been disassociated.")
         failure_message _("Could not disassociate the environment")
 
-        extend_with(HammerCLIForeman::CommandExtensions::PuppetEnvironment.new)
```

Comment:
should i remove line such as:  extend_with(HammerCLIForeman::CommandExtensions::PuppetEnvironment.new) in general? @ofedoren

Reply:
@adiabramovitch, please don't :) This line allows us to use that extension which still should have custom `option_sources`.